device/mellanox/x86_64-nvidia_sn4280_simx-r0/ACS-SN4280/qos.json.j2

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/qos.json.j2
```

Comment:
For the smart switch, any difference between the FP ports and the ports connected to nics in terms of qos?